### DISCUSSION TITLE
How to convert DFT OUTCAR to data used in maml

### DISCUSSION THREAD

**Discussion by WJiangH** [materialsvirtuallab/maml#582] [Q&A] [answered]
Hi,

Is there any build-in functions in maml that can convert directly the DFT results into the desire data format used in maml training or testing?

Thanks.

Best,
JJ

**Comment by shyuep** [comment, marked as answer]
We generally do not use OUTCARs, but vasprun.xml. You can look at pymatgen to find out how to convert them.
